apps/src/code-studio/components/progress/teacherPanel/TeacherPanel.jsx

```diff
@@ -121,7 +121,9 @@ class TeacherPanel extends React.Component {
       viewAs === ViewType.Teacher && currentStudent;
 
     const displayLevelExamples =
-      viewAs === ViewType.Teacher && sectionData && sectionData.level_examples;
+      viewAs === ViewType.Teacher &&
+      sectionData &&
+      sectionData.level_examples?.length > 0;
```

Comment:
You can even do multiple layers of optional chaining like `sectionData?.level_examples?.length > 0`